pkg/util/flags/flags.go

```diff
@@ -81,6 +82,14 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
+func SetFlagsFromCmdline(flags *pflag.FlagSet) {
```

Comment:
What if we call this `SetLoggingFlags()`? This seems specific to logging.

Reply:
Sure. that will work too.